### PR TITLE
chore: update mongodb option object

### PR DESCRIPTION
**Pull request by labajo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "alfalfa",
-  "version": "2.2.0",
+  "version": "2.2.1",
```
Comment (palmerabollo) [inline package.json]: I think this should be 3.0.0 because you are breaking backwards compatibility.

Comment (jmendiara) [inline package.json]: Agree. Also, a feature PR *should* not include a version bump

**Comment by calamarico** [comment]
tumbleweed

**Comment by jorgems101** [comment]
Hello, What is the status of this PR? To merge it we only need change version to 3.0.0 due to backwards compatibility?

**Comment by palmerabollo** [comment]
Yes, I merged the changes but I was not able to publish a new 3.0.0 release since it's raising some typescript errors (unrelated to @labajo's changes I think). I @labajo can't compile it I think we need @jmendiara

**Comment by jorgems101** [comment]
Thank you very much @palmerabollo :-) I hope @jmendiara can help us with this.

**Comment by jmendiara** [comment]
The dependencies are outdated, and the lack of dependencies locking is causing compile errors. 
It will take a while to solve them

**Pull request merged**